docs/developing/ui-component-testing.md

```diff
@@ -0,0 +1,32 @@
+# UI Component Testing
+
+This document gives you a process to follow to check that any UI components
+you have created will be in tip-top shape for the site.
+
+## Browsers to check
+
+It doesn't have to look exactly the same in every browser, but it should look similar and work properly in current versions of these evergreen browsers:
+
+### Desktop
+
+* Firefox
+* Opera
```

Comment:
Is it really worth testing in Opera considering it uses the same Blink engine as Chrome and it has such a small user bases?

Reply:
@davidcmoulton ...?